katas/content/multi_qubit_measurements/measuring_one.qs

```diff
@@ -1,6 +1,6 @@
 namespace Kata {
     open Microsoft.Quantum.Diagnostics;
-    open Microsoft.Quantum.Arithmetic;
+    open Microsoft.Quantum.Measurement;
```

Comment:
This change would need to be deployed to the katas site, which we should coordinate to make sure there isn't a difference between katas namespaces and what VSCode.dev supports.

Reply:
Cesar says we'll coordinate once this is in.